### PR TITLE
Document running Toil on Google Cloud (resolves #889)

### LINKED ISSUE #889: Documentation on running using Google job store
We need some documentation in the docs section: 
http://toil.readthedocs.io/en/latest/installation.html#cloud-installation

And in the index mentioning support. We can mention its a prototype feature. 

I think this is urgent for the paper - I should have noted it before.

### PR DESCRIPTION
**Pull request by cket**
Documents both status of support for running on Google Cloud Engine and how to set up your Google account to work with the Google job store.

### CODE REVIEW

File: docs/running.rst [hunk 1]
```diff
@@ -384,8 +384,38 @@ Note that once you run a job with a particular job store name (the part after th
...
+
+    [Boto]
+    https_validate_certificates = True
+
+    [GSUtil]
+    content_language = en
+    default_api_version = 2
+
+The gs_access_key_id and gs_secret_access_key can be generated by navigating to your Google Cloud Storage console and clicking on 'Settings'.
+Then, on the Settings page, navigate to the Interoperability tab and click 'Enable interoperability access'. On this page you can now click 'Create a new key' to generate and access key and a matching secret.
```
Comment (hannes-ucsc) [inline docs/running.rst]: `generate an` not `generate and`

File: docs/installation.rst [hunk 2]
```diff
@@ -298,3 +298,13 @@ Installation on OpenStack
...
 to run Toil in **singleMachine** mode following the basic installation instructions: :ref:`installation-ref`
+
+.. _installationGoogleComputeEngine:
+
+Installation on Google Compute Engine
+-------------------------------------
+
+Support for running on Google Cloud is experimental, and our group is working to expand
+distributed cluster support to Google Compute by writing a cluster provisioning tool based around
+a Dockerized Mesos setup. Currently, Google Compute Engine nodes can be configured to
```
Comment (hannes-ucsc) [inline docs/installation.rst]: `dockerized` or `Docker-ized`

File: docs/running.rst [hunk 3]
```diff
@@ -384,8 +384,38 @@ Note that once you run a job with a particular job store name (the part after th
...
+Insert these into their respective places in the .boto file and you will be able to use a Google job store when invoking a Toil script, as in the following example::
+
+    python HelloWorld.py google:projectID:jobStore
```
Comment (hannes-ucsc) [inline docs/running.rst]: Not clear that `jobStore` is to be chosen by the user.

File: docs/installation.rst (hunk 2 above)
Comment (jvivian) [inline docs/installation.rst]: Docker's official docs use: Dockerized
https://docs.docker.com/engine/userguide/containers/dockerizing/

Comment (hannes-ucsc) [inline docs/installation.rst]: I don't care. It's a fucking verb.

**Pull request merged**